src/content/docs/tutorial-errors/respond-outages.mdx

```diff
@@ -0,0 +1,155 @@
+
+Now that your apps are instrumented, New Relic is capturing data about your services. This includes data about error occurrences in your app. 
+
+<Steps>
+    <Step>
+        ### Think about the end user
+
+Alerts let you know a problem exists: they're the water on your lawn. But alerts won't provide you all the context. That's where your errors inbox comes in. 
+
+Imagine you're responsible for some apps on an ecommerce site. You've received two alerts for two components, one for checking out and one for searching inventory. You're only receiving reports that the search functionality is failing for end users, but the check out component works fine. You may believe that the check out function is more important, but it's critical to separate your beliefs from your observability practices. 
+
+This practice applies even if the end user hasn't reported anything. When you notice services failing, you can ask yourself these questions:
+
+* Is the end user experiencing a problem? 
+* How should their experience look?
+* What behavior are they currently experiencing?
+
+    </Step>
+    <Step>
+        ### Choose a service
+
+Let's see how this might look in practice. When you view the **All entities** page, you notice four services are alerting.
+
+<img
+title="Overview errors affecting your services"
+alt="A screenshot showing an app with many errors"
+src={allEntitiesView}
+/>
+
+After asking yourself the questions from step one, you know that: 
+
+* The end user is struggling with purchase actions.
+* Your site should only display in-stock items.
+* Your site is displaying all products, so customers are able to purchase out-of-stock items.
+
+After performing your audit, you know that `api-gateway` is a critical dependency for your inventory. This is where you'll begin your error analysis. 
+
+    </Step>
+    <Step>
+
+### Locate what changed [#source] 
+
+You have your entry point into your system, so now you can look into the errors affecting your app. From the `api-gateway` summary page, click the **Errors** tab under **Triage**. Your errors page filters your data to an errors-only view. 
+
+<img
+title="Overview errors affecting your services"
+alt="A screenshot showing an app with many errors"
+src={apmErrorsInboxPage}
+/>
+
+`api-gateway` has at least six error groups reporting with anywhere from a dozen to thousands of occurrences in your app. Don't make any assumptions about what error is critical. Instead, start with the time series on the far right of the waterfall view. 
+
+At first this seems to lack granularity, but your time series gives you enough information to point to what changed over time. We'll break this down:
+
+* Based on number of occurrences alone, your first instinct might tell you to start with `ActivemModel:::ValidationError` as it has 4,000 occurrences. If you look at the time series, though, its peaks and troughs are relatively consistent. This could be an expected error, but let's look at the other five. 
```

Comment:
Similar to my point about sounding pedantic above, perhaps streamline? My worry is that we're setting up so many "You think it's like X but it's actually like Z" moments that it's like a thriller with too many twists. This could also be that it's all "you"; which implicitly makes the reader feel fooled or like they're making a mistake.

What about:

> The `ActivemModel:::ValidationError` error looks like a good candidate, with over 4,000 occurrences! But as we look at the time series, its peaks and troughs are relatively consistent. This could be an expected error, so let's look at the other five.

Reply:
I don't want to over-rotate on this; your storytelling is very cool throughout this doc.